dev/com.ibm.websphere.appserver.features/visibility/public/websocket-1.1/com.ibm.websphere.appserver.websocket-1.1.feature

```diff
@@ -10,10 +10,9 @@ IBM-API-Package: javax.websocket; version="1.1"; type="spec", \
  com.ibm.websphere.wsoc; type="ibm-api"
 IBM-ShortName: websocket-1.1
 Subsystem-Name: Java WebSocket 1.1
--features=com.ibm.websphere.appserver.servlet-3.1; ibm.tolerates:="4.0", \
-  com.ibm.websphere.appserver.eeCompatible-7.0; ibm.tolerates:="6.0,8.0"
```

Comment:
Why remove the `eecompatible` feature here?

Reply:
By including in the private feature and pulling in a single version of the private feature, it isn't necessary to include it in both public and private. When tolerating more versions of the private feature with different eeCompatible versions, then we would likely want to explicitly include it in the public feature as well. Since we only pull in one version of the private feature and dont tolerate any other we should be good to go

Reply:
While i agree that at the moment it _should_ work without eeCompatible, I think it would be better to leave it set in the public feature as well. I think that at the very least, keeping it in the public feature will make the potential error message slightly shorter (a shorted dependency chain). @jhanders34 and @tjwatson should be able to confirm.

Reply:
In general I tried to have eeCompatible in ALL of the EE public features so that it was obvious to the user which feature combination caused the conflict.  So if you are removing it in this PR, I agree with Tom that it should be put back.

Reply:
Ok sounds good Ill make the change to add it again.